platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -18,6 +18,8 @@ class ViewerRetrieveStudyData extends Component {
 
   constructor(props) {
     super(props);
+    this.abortSeriesLoad = false;
+    this.seriesLoadStats = Object.create(null);
```

Comment:
An object, where each property is a study Uid?

Reply:
yes!